adagrams/game.py

```diff
@@ -1,13 +1,132 @@
 from random import randint
 
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+HAND_SIZE = 10
+PILE_SIZE = len(LETTER_POOL)
+
+def make_new_draw_pile():
+    draw_pile = []
+    for letter, count in LETTER_POOL.items():
+        [draw_pile.append(letter)] * count
+    return draw_pile
+
 def draw_letters():
-    pass
+    new_pile = make_new_draw_pile()
+    new_pile_len = PILE_SIZE
+    hand = []
+
+    for _ in range(HAND_SIZE):
+        hand.append(new_pile.pop(randint(0, new_pile_len - 1)))
+        new_pile_len -= 1
+
+    return hand
```

Comment:
👀 The logic in `draw_letters` works correctly (see notes about PILE_SIZE), however, due to an issue in the `make_new_draw_pile` helper, the hands will only ever contain a single copy of any given tile.